website/content/community/maintainers.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Maintainers
+weight: 1
+---
+
+MetalLB was created by Dave Anderson.  We are thankful for his efforts to
+create MetalLB as a valuable contribution to the Kubernetes ecosystem.  Ongoing
+MetalLB development and maintenance is run by a team of current maintainers.
```

Comment:
```suggestion
MetalLB development and maintenance is done by a team of maintainers.
```

- "development is run" sounds strange to me.
- I find "current maintainers" confusing. I suggest we simply say "a team of maintainers" (or explain what we mean by "current").

Reply:
is "led", perhaps?  I imagine developing being done by a larger community, but led (or run, organized, something) by the maintainers team.

I've included your suggested text for now though

Reply:
"led" SGTM.

Reply:
changed to "led"